CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 2.0.0-beta.13 - June 4, 2021
+
+For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
+
+### Changelog
+#### Bug fixes and improvements
+* Removed `getRouteGeometryWithBuffer` API from `RouteProgress`. [#4478](https://github.com/mapbox/mapbox-navigation-android/pull/4478)
+* Removed the `updatePrimaryRoute` method from the `MapboxRouteLineApi` class. It was replaced with an extension function `RouteLineUtils::updatePrimaryRoute` which reorders a collections of `DirectionsRoute` objects. [#4467](https://github.com/mapbox/mapbox-navigation-android/pull/4467)
+* Recreate Navigator to switch tiles version. [#4451](https://github.com/mapbox/mapbox-navigation-android/pull/4451)
```

Comment:
Could you elaborate here a little bit? Let's at least mention what this means and link to the interface.

Reply:
How about?

> Added `NavigationVersionSwitchObserver` which enables listening to navigation tiles version switch.